PROTOCOL.md

```diff
@@ -1174,12 +1174,15 @@ Example:
 
 This is the API for clients to get a table version without any other extra information. The server usually can implement this API effectively. If a client caches information about a shared table locally, it can store the table version and use this cheap API to quickly check whether their cache is stale and they should re-fetch the data.
 
+Note: We are migrating this method from HEAD to GET, and with a `/version` suffix. Please use the new API as we'll deprecate the support of HEAD API soon. 
```

Comment:
It's an open source project. "we'll deprecate the support of HEAD API soon" does not make sense in this context...

Rather, you should specify a timeline when this will no longer be part of the protocol so that both clients and servers can follow them. In fact, we may want to add somewhere about expected timelines for backwards-incompatible changes.